flaml/tune/tune.py

```diff
@@ -96,6 +96,8 @@ def compute_with_config(config):
             _running_trial = trial
         result["training_iteration"] = _training_iteration
         result["config"] = trial.config
+        if "incumbent_result" in result["config"].keys():
+            del result["config"]["incumbent_result"]

```

Comment:
```suggestion
        if INCUMBENT_RESULT in result["config"]:
            del result["config"][INCUMBENT_RESULT ]
```
Define the constant INCUMBENT_RESULT in the beginning.
Question: Is it necessary to delete it?